deploy/default-azuredeploy.json

```diff
@@ -36,9 +36,10 @@
       "type": "string",
       "allowedValues": [
         "v1.0.0",
-        "v2.0.0"
+        "v2.0.0",
+        "latest"
       ],
-      "defaultValue": "v2.0.0"
+      "defaultValue": "latest"
```

Comment:
Does thee `latest` tag refer to the latest commit in master branch?

Reply:
Yes.

Reply:
I think we'd better use a stable version as default? It seems a bit risky to fetch latest commit from master as default.

Reply:
Perhaps we could release another v2.0.1 or v2.1.0?

Reply:
Sure, I believe a straightforward tag is more appropriate.